pylint/checkers/variables.py

```diff
@@ -607,6 +616,25 @@ def get_next_to_consume(self, node):
                 or n.statement().parent_of(node)
             ]
 
+        # Filter out assignments in an Except clause that node is not
+        # contained in, assuming they may fail
+        if found_nodes:
+            filtered_nodes = [
+                n
+                for n in found_nodes
+                if not (
+                    isinstance(n.statement().parent, nodes.ExceptHandler)
```

Comment:
@jacobtylerwalls Sorry about that. Seems like I forgot to actually publish my review..

Comment:
Could you use `statement(future=True)` here and on the other lines? The current behaviour of `statement()` is being deprecated and the `future` keyword can be used to trigger the future behaviour.